pkg/server/version_cluster_test.go

```diff
@@ -414,6 +414,7 @@ func TestClusterVersionMixedVersionTooOld(t *testing.T) {
 	knobs := base.TestingKnobs{
 		Server: &server.TestingKnobs{
 			DisableAutomaticVersionUpgrade: make(chan struct{}),
+			BootstrapVersionKeyOverride:    clusterversion.BinaryMinSupportedVersionKey,
```

Comment:
Is it okay that the binary version is going to be lower than the bootstrap version? From looking at the definition of `v0v1` above, it seems that `v0` is going to be one version lower than `clusterversion.BinaryMinSupportedVersionKey`. I guess ideally it would use `v0` as the bootstrap version but it seems like that version might either not exist (the code in `v0v1` seems to allow `0` as the minor version) or be deleted already.

Reply:
Simplified the test to use `TestingBinaryMinSupportedVersion` and `TestingBinaryVersion ` instead.